symplyphysics/laws/hydro/excessive_pressure_under_curved_surface_of_bubble.py

```diff
@@ -0,0 +1,53 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+from symplyphysics.laws.thermodynamics import laplas_pressure as laplas_law
+
+# Description
+## Under the curved surface of the liquid, in addition to the internal pressure,
+## additional pressure is created due to the curvature of the surface.
+## For a bubble, this pressure is created by two surfaces: the outer and the inner.
+
+## Law is: p = 4 * sigma / R, where
+## p - excessive pressure under the curved surface of bubble,
+## sigma - surface tension of the liquid,
+## R - radius of bubble.
+
+excessive_pressure = Symbol("excessive_pressure", units.pressure)
+
+surface_tension_of_the_liquid = Symbol("surface_tension_of_the_liquid", units.force / units.length)
+radius_of_bubble = Symbol("radius_of_bubble", units.length)
+
+law = Eq(excessive_pressure, 4 * surface_tension_of_the_liquid / radius_of_bubble)
+
+# This law might be derived via Laplas law.
+# The bubble has two surfaces – an outer and an inner one, each of which creates additional pressure.
+# Therefore, an additional multiplier of "2" appears.
+
+laplas_law_applied = laplas_law.law.subs({
+    laplas_law.surface_tension_of_the_liquid: 2 * surface_tension_of_the_liquid,
```

Comment:
No, it does not have double tension.

Reply:
> No, it does not have double tension.

I wrote an explanation on top that tells you the reason for the appearance of the two.

So that it doesn't look like a change in surface tension, I can just add a two like this:
assert expr_equals(2 * pressure_derived, law.rhs)

Reply:
You should calculate pressure from laplace, write that it is for semisphere and bubble had two of them hence double pressure.